WebRoot/public/error.jsp

```diff
@@ -29,7 +29,11 @@
 
 <%
 	String skin = request.getParameter("skin");
-	if (skin == null || !Pattern.matches("^[0-9A-Za-z]+$", skin)) {
+	if (skin == null) {
+		Object o = pageContext.getAttribute("skin", PageContext.REQUEST_SCOPE);
+		skin = String.valueOf(o);
+	}
+	if (skin == null || "null".equals(skin) || !Pattern.matches("^[0-9A-Za-z]+$", skin)) {
 		skin = application.getInitParameter("zimbraDefaultSkin");
```

Comment:
I am a little bit confused on why we need this here, skinAndRedirect is already handling logic of getting correct skin parameter so why do we need to get zimbraDefaultSkin here?

Reply:
The logic has been added on https://github.com/Zimbra/zm-web-client/pull/578
In `error.jsp`, a skin can be specified using a url parameter like `https://host/aaa?skin=xxx`
If `skin` parameter is specified but the value does not match `^[0-9A-Za-z]+$`, `zimbraDefaultSkin` is set as a skin here.

The 2nd logic being added in the PR (which is described in Description of the PR) is executed only when a request url does not have `skin` parameter.